exporter/clickhouselogsexporter/exporter.go

```diff
@@ -240,7 +266,15 @@ func (e *clickhouseLogsExporter) pushToClickhouse(ctx context.Context, ld plog.L
 
 			resources := attributesToSlice(res.Attributes(), true)
 
-			err := addAttrsToTagStatement(tagStatement, "resource", resources)
+			resourcesMap := attributesToMap(res.Attributes(), true)
+
+			serializedRes, err := json.Marshal(res.Attributes().AsRaw())
+			if err != nil {
+				return fmt.Errorf("couldn't serialize log resource JSON: %w", err)
+			}
+			resourceJson := string(serializedRes)
```

Comment:
This is not guaranteed to produce a consistent json string even for the same set of attributes if their order of insertion is not the same in map. What happens if the `ResourceLogs` has multiple entries with same attrs but different order?